config/application.rb

```diff
@@ -37,13 +37,16 @@ class Application < Rails::Application
     # config.active_record.raise_in_transactional_callbacks = true
 
     # Custom directories with classes and modules you want to be autoloadable.
-    config.autoload_paths += Dir["#{config.root}/lib", "#{config.root}/lib/**/"]
-    config.eager_load_paths += Dir["#{config.root}/lib", "#{config.root}/lib/**/"]
+    # config.autoload_paths += Dir["#{config.root}/lib", "#{config.root}/lib/**/"]
```

Comment:
I mean I don't really care since this is code is really not going to be expanded further but, can we just remove these old ones or leave a comment